src/components/ListLayout.jsx

```diff
@@ -18,41 +19,113 @@ const deleteItemFromDb = async (token, itemId) => {
 
 const ListLayout = ({ items, localToken }) => {
   const [filter, setFilter] = useState('');
+  const [layoutItems, setLayoutItems] = useState(items);
+
+  const [checkedItems, setCheckedItems] = useState([]);
 
-  const currentTime = Date.now();
   const oneDay = 86400000; //24 hours in milliseconds
   //create a reference for an input
+  const currentTime = Date.now();
   const inputRef = useRef(null);
 
   useEffect(() => {
     inputRef.current.focus();
   }, []);
 
+  useEffect(() => {
+    items.forEach((item) => {
+      // updates isActive property of item to true if item has 2+ purchases and has been purchased within calculated estimate
+      // isActive is defaulted to false when item is added
+      const dateOfLastTransaction =
+        item.totalPurchases > 0 ? item.purchasedDate : item.createdAt;
+      const daysSinceLastTransaction =
+        (currentTime - dateOfLastTransaction) / oneDay;
```

Comment:
Another PR is currently working on adding constants for time, including this one. Maybe create new issue in GitHub to use that here and remove oneDay once this PR and other PR are submitted?